.github/workflows/ci.yml

```
@@ -214,6 +214,18 @@ jobs:
         with:
           mdbook-version: '0.4.17'
 
+  mdbook-lint:
+    needs: cancel-previous-runs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - run: |
+          npm install -g markdownlint-cli@https://github.com/igorshubovych/markdownlint-cli/archive/refs/tags/v0.31.1.tar.gz
```

Does

```suggestion
          npm install -g markdownlint-cli@0.31.1
```

not work?

---

@adlerjohn It most certainly does (I think this is a holdover from some debugging I was doing). Change pushed